configdb/hardware/models.py

```diff
@@ -204,15 +221,15 @@ def __str__(self):
 
 
 class ModeType(BaseModel):
-    id = models.CharField(max_length=200, primary_key=True)
+    id = models.CharField(max_length=200, primary_key=True, help_text='Mode type ID')
```

Comment:
This is technically a string name/code of the mode type. It is intended to be a "code" or "id" I guess in that we don't want spaces in it. Maybe just call it the "Mode type" rather than "Mode type ID"?